sonic_y_cable/y_cable.py

```diff
@@ -8,9 +8,10 @@
     import struct
     from sonic_py_common import logger
     import sonic_platform.platform
+    import math
```

Comment:
Please reorganize imports per PEP8 standards and alphabetize:

```python
    import math
    import struct

    import sonic_platform.platform
    from sonic_py_common import logger
```

Reply:
fixed